Resources/public/Admin.js

```diff
@@ -342,9 +342,9 @@ var Admin = {
             var collection = jQuery(this);
             var counter = 0;
             collection.children().each(function() {
-                var matches = highestCounterRegexp.exec(jQuery(this).find('.form-control').attr('id'));
+                var matches = highestCounterRegexp.exec(jQuery('[id^="sonata-ba-field-container"]', this).attr('id'));
                 if (matches && matches[1] && matches[1] > counter) {
-                    counter = matches[1];
+                    counter = parseInt(matches[1]);
```

Comment:
Yes of course :)

On Mon, Apr 6, 2015, 15:18 Sullivan SENECHAL <notifications@github.com>
wrote:

> In Resources/public/Admin.js
> <https://github.com/sonata-project/SonataAdminBundle/pull/2889#discussion_r27796105>
> :
>
> >                  if (matches && matches[1] && matches[1] > counter) {
> > -                    counter = matches[1];
> > +                    counter = parseInt(matches[1]);
>
> Interesting. Ok for that but matches[1] should be parsed instead of
> matches, isn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sonata-project/SonataAdminBundle/pull/2889/files#r27796105>
> .
>